site/docs/v1/tech/email-templates/templates-replacement-variables.adoc

```diff
@@ -520,6 +522,110 @@ The Tenant object, see the link:/docs/v1/tech/apis/tenants[Tenant API] for field
+    <br>
+    <strong>Identifier: ${method.id}</strong>
+
+</p>
+
+- FusionAuth Admin
+----
+
+.Text
+[source,text]
+----
+The following two factor method was added to ${user.email}:
+
+- Method: ${method.method}
+- Identifier: ${method.id}
+
+- FusionAuth Admin
+----
+
+=== Replacement Variables
+
+[.api]
+[field]#application# [type]#[Application]# [since]#Available since 1.28.0#::
+The Application object, see the link:/docs/v1/tech/apis/applications[Application API] for field definitions.
++
+*Note*:
+This object may not be available depending upon when this template is constructed. If you utilize this object in your template, ensure you first check to see if it is defined.
+
+[field]#event# [type]#[Event]#::
+The Event object for a two factor add event.  See the link:/docs/v1/tech/events-webhooks/events/user-two-factor-method-add[Webhooks & Events section] for field definitions.
+
+[field]#method# [type]#[Object]#::
+The two-factor method that was added. See the link:/docs/v1/tech/apis/two-factor[Multi Factor/Two Factor APIs] for property definitions and example JSON.
+
+[field]#tenant# [type]#[Tenant]#::
+The Tenant object, see the link:/docs/v1/tech/apis/tenants[Tenant API] for field definitions.
+
+[field]#user# [type]#[User]#::
+The User object, see the link:/docs/v1/tech/apis/users[User API] for field definitions of a User.
+
+== Two Factor Authentication Method Removed
+
+include::docs/v1/tech/shared/_enterprise-edition-blurb.adoc[]
+
+.HTML
+[source,text]
+----
+<p>
+The following two factor was removed from ${user.email}:
+
+    <br>
+    <strong>Method: ${method.method}</strong>
+    <br>
+    <strong>Identifier: ${method.id}</strong>
+
+</p>
+
+- FusionAuth Admin
+----
+
+.Text
+[source,text]
+----
+The following two factor method was removed from ${user.email}:
+
+- Method: ${method.method}
+- Identifier: ${method.id}
+
+- FusionAuth Admin
+----
+
+=== Replacement Variables
+
+[.api]
+[field]#application# [type]#[Application]#::
+The Application object, see the link:/docs/v1/tech/apis/applications[Application API] for field definitions.
++
+*Note*:
+This object may not be available depending upon when this template is constructed. If you utilize this object in your template, ensure you first check to see if it is defined.
+
+[field]#event# [type]#[Event]#::
+The Event object for a two factor remove event.  See the link:/docs/v1/tech/events-webhooks/events/user-two-factor-method-remove[Webhooks & Events section] for field definitions.
+
+[field]#method# [type]#[Object]#::
+The two-factor method that was added. See the link:/docs/v1/tech/apis/two-factor[Multi Factor/Two Factor APIs] for property definitions and example JSON.
+
+[field]#tenant# [type]#[Tenant]#::
+The Tenant object, see the link:/docs/v1/tech/apis/tenants[Tenant API] for field definitions.
+
+[field]#user# [type]#[User]#::
+The User object, see the link:/docs/v1/tech/apis/users[User API] for field definitions of a User.
+
 == Threat Detected
```

Comment:
Why is this entry below the two factor add/remove in the text, but above it in the TOC? Please sync those up.